src/main/java/org/opensearch/performanceanalyzer/PerformanceAnalyzerApp.java

```diff
@@ -51,6 +53,7 @@ public class PerformanceAnalyzerApp {
 
     private static final Logger LOG = LogManager.getLogger(PerformanceAnalyzerApp.class);
 
+    public static final int READER_RESTART_MAX_ATTEMPTS = 3;
```

Comment:
We should probably do some test runs in the docker env to understand what could be the correct value for `READER_RESTART_MAX_ATTEMPTS`. We can also consider exponential backoff here in case the env set up takes time.

Reply:
I will increase the number of attempts here to 12(1min). 

Exponential Backoff makes sense for API calls but here, a thread is crashing while attempting to read data from disk - unlikely, the backoff retry will help here.